modules/book.js

```diff
@@ -0,0 +1,44 @@
+class BookCollection {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+    this.addBook = this.addBook.bind(this);
+    this.removeBook = this.removeBook.bind(this);
+  }
+
+  addBook() {
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    if (title && author) {
+      this.books.push({ title, author });
+      localStorage.setItem('books', JSON.stringify(this.books));
+      this.displayBooks();
+      document.getElementById('title').value = '';
+      document.getElementById('author').value = '';
+    }
+  }
+
+  removeBook(index) {
+    this.books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.displayBooks();
+  }
+
+  displayBooks() {
+    const bookList = document.getElementById('books');
+    bookList.innerHTML = '';
+    this.books.forEach((book, index) => {
+      const li = document.createElement('li');
+      const h3 = document.createElement('h3');
+      const removeButton = document.createElement('button');
+      h3.textContent = `${book.title} by ${book.author}`;
+      removeButton.textContent = 'Remove';
+      removeButton.addEventListener('click', () => this.removeBook(index));
+      li.classList.add('book-item');
+      li.appendChild(h3);
+      li.appendChild(removeButton);
+      bookList.appendChild(li);
+    });
+  }
+}
```

Comment:
- [ ] Great job so far :+1: . According to the project requirements, you are to refactor your functions to arrow functions but that hasn't been implemented in this part of your codebase. So kindly include that requirement in your project :wink: 
- Project Requirement
![image](https://user-images.githubusercontent.com/102699958/220165502-ba047693-016a-4313-a2c9-ea081d221d26.png)

Reply:
Hello, @mumo-esther I was notified that you were only asked to refactor functions to arrow functions and not class methods. So you can skip this comment and work on the other ones. My apologies for this :relaxed: